app/controllers/translated_attributes_controller.rb

```diff
@@ -56,8 +56,8 @@ def load_translated_attr
   end
 
   def permitted_params
-    data_attrs[:onesky_phrase_id] = data_attrs.delete(:"onesky-phrase-id") if data_attrs[:"onesky-phrase-id"]
-    permit_params(:key, :onesky_phrase_id, :required)
+    data_attrs[:crowdin_phrase_id] = data_attrs.delete(:"crowdin-phrase-id") if data_attrs[:"crowdin-phrase-id"]
+    permit_params(:key, :crowdin_phrase_id, :required)
```

Comment:
You might excise "onesky" / "OneSky" / "ONESKY" from some/all of the following as well:
- README.md
- app/lib/package.rb
- app/models/xml/translatable.rb
- public/xmlns/content.xsd
- schema_tests/manifest/valid/tools/satisfied.xml
- spec/models/translation_spec.rb